addons/binding/org.openhab.binding.vera/src/main/java/org/openhab/binding/vera/handler/VeraDeviceHandler.java

```diff
@@ -0,0 +1,468 @@
+                            break;
+                        case 4:
+                            id = SENSOR_SMOKE_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                        case 5:
+                            id = SENSOR_CO_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                        case 6:
+                            id = SENSOR_BINARY_CHANNEL;
+                            acceptedItemType = "Switch";
+                            break;
+                    }
+                    break;
+                case HVAC: // TODO
+                    logger.warn("TODO: {}, {}", device, device.getCategoryType());
+                    break;
+                case DoorLock:
+                    id = DOORLOCK_CHANNEL;
+                    acceptedItemType = "Switch";
+                    tag = "Switchable";
+                    break;
+                case WindowCovering:
+                    id = SWITCH_ROLLERSHUTTER_CHANNEL;
+                    acceptedItemType = "Rollershutter";
+                    break;
+                case GenericSensor:
+                    id = SENSOR_BINARY_CHANNEL;
+                    acceptedItemType = "Switch";
+                    break;
+                case SceneController:
+                    id = SWITCH_BINARY_CHANNEL;
+                    acceptedItemType = "Switch";
+                    break;
+                case HumiditySensor:
+                    id = SENSOR_HUMIDITY_CHANNEL;
+                    acceptedItemType = "Number";
+                    tag = "CurrentHumidity";
+                    break;
+                case TemperatureSensor:
+                    id = SENSOR_TEMPERATURE_CHANNEL;
+                    acceptedItemType = "Number";
+                    tag = "CurrentTemperature";
+                    break;
+                case LightSensor:
+                    id = SENSOR_LUMINOSITY_CHANNEL;
+                    acceptedItemType = "Number";
+                    break;
+                case PowerMeter:
+                    id = SENSOR_ENERGY_CHANNEL;
+                    acceptedItemType = "Number";
+                    break;
+                case UVSensor:
+                    id = SENSOR_ULTRAVIOLET_CHANNEL;
+                    acceptedItemType = "Number";
+                    break;
+                case Camera:
+                case RemoteControl:
+                case IRTransmitter:
+                case GenericIO:
+                case SerialPort:
+                case AV:
+                case ZWaveInterface:
+                case InsteonInterface:
+                case AlarmPanel:
+                case AlarmPartition:
+                case Siren:
+                case Weather:
+                case PhilipsController:
+                case Appliance:
+                    logger.warn("TODO: {}, {}", device, device.getCategoryType());

```

Comment:
Can you change it to `debug` instead? I wouldn't like a log file full of warnings just because devices aren't yet supported by a binding. It would also help to make a note about this in the docs.